posts/2021-11-09-now-with-more-elixir.md

```diff
@@ -0,0 +1,208 @@
+%{
+author: "Sean Callan",
+author_link: "https://github.com/doomspork",
+tags: ["Announcement"],
+date: ~D[2021-11-09],
+title: "Now With More Elixir!",
+excerpt: """
+How Elixir School migrated from Jekyll to a Phoenix powered site
+"""
+}
+
+---
+
+# Now With More Elixir!
+
+If you follow us on Twitter then you may have gotten a sneak peek at the new site before today's big launch but for those who haven't followed along we thought we'd cover this exciting new chapter in our history.
+
+## TL;DR The History
+
+The very first commit for Elixir School took place the day after my birthday many moons ago now: May 31, 2015. Since then the project has evolved from me to over 500 individuals who've contributed new content, translations, and other improvements. As things grew it became clear we needed to migrate from markdown in a repo, and given the lack of options in the Elixir space at the time, we selected Jekyll. Jekyll got us online and continues to serve us well provide a static site that handles 10s of thousands of weekly users all across the globe.
```

Comment:
`continues to serve us well provide a static site` sounds a bit weird to me but maybe it's right? Shouldn't it be `in providing` or `to provide`? Not sure.

Reply:
I think it should be either:
`Jekyll got us online and served us well to provide a static site that handled 10s of thousands of weekly users across the globe.` If it is meant to be past tense, or:

`Jekyll got us online and continues to serve us well to provide a static site that handles 10s of thousands of weekly users across the globe.` If it is meant to be present tense.

Does Elixir School still use Jekyll?

Reply:
> Does Elixir School still use Jekyll?

Nope!

Reply:
```suggestion
The very first commit for Elixir School took place the day after my birthday many moons ago now: May 31, 2015. Since then the project has evolved from me to over 500 individuals who've contributed new content, translations, and other improvements. As things grew it became clear we needed to migrate from markdown in a repo, and given the lack of options in the Elixir space at the time, we selected Jekyll. Jekyll got us online and served us well to provide a static site that handled 10s of thousands of weekly users across the globe.
```